src/Interpreters/TranslateQualifiedNamesVisitor.cpp

```diff
@@ -249,7 +249,20 @@ void TranslateQualifiedNamesMatcher::visit(ASTExpressionList & node, const ASTPt
                     for (const auto & column : *cols)
                     {
                         if (first_table || !data.join_using_columns.contains(column.name))
-                            addIdentifier(columns, table.table, column.name);
+                        {
+                            std::string column_name = column.name;
+                            std::string::size_type pos = 0u;
+                            for (auto parameter : data.parameter_values)
```

Comment:
why not make like this?
```suggestion
                            for (const auto & parameter : data.parameter_values)
```

Reply:
addressed it